client/my-sites/media-library/list-no-content.jsx

```diff
@@ -18,7 +18,12 @@ class MediaLibraryListNoContent extends Component {
 
 		//TODO: handle each service with individual messages
 		if ( 'google_photos' === source ) {
-			return translate( "You don't have any photos in your Google library.", {
+			if ( 'videos' === filter ) {
+				return translate( "You don't have any videos in your Google Photos library.", {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.